Branch-SDK/src/io/branch/referral/BranchViewHandler.java

```diff
@@ -138,36 +129,82 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
                     if (!isHandled) {
                         view.loadUrl(url);
                     } else {
-                        dialog.dismiss();
+                        if (branchViewDialog_ != null) {
+                            branchViewDialog_.dismiss();
+                        }
                     }
                     return isHandled;
                 }
 
                 @Override
+                public void onPageStarted(WebView view, String url, Bitmap favicon) {
+                    super.onPageStarted(view, url, favicon);
+                }
+
+                @Override
+                public void onReceivedError(WebView view, int errorCode, String description, String failingUrl) {
+                    super.onReceivedError(view, errorCode, description, failingUrl);
+                    webViewLoadError_ = true;
```

Comment:
You said this happened regularly? Do we need to handle dismissal and callback to the dev when this occurs?